ukis_pysat/raster.py

```diff
@@ -63,10 +59,7 @@ def __init__(self, data, dimorder="first", crs=None, transform=None):
             else:
                 self.__arr = reshape_as_raster(data)
             meta = {"dtype": self.__arr.dtype, "count": self.__arr.shape[0], "crs": crs, "driver": "GTiff"}

```

Comment:
is it possible that crs is redundant here? could remove it in meta dict.

Reply:
Maybe meta is redundant, I'll check.